WordPress/Classes/ViewRelated/Views/List/NotificationsList/HostingTableViewCell.swift

```diff
@@ -0,0 +1,27 @@
+import UIKit
+import SwiftUI
+
+class HostingTableViewCell<Content: View>: UITableViewCell {
+    private weak var controller: UIHostingController<Content>?
+
+    func host(_ view: Content, parent: UIViewController) {
+        if let controller = controller {
+            controller.rootView = view
+            controller.view.layoutIfNeeded()
```

Comment:
[Discussion] I'm not sure if you ran into an issue where the cell doesn't resize to fit its content. From experience, the only method that worked for me is `controller.view.invalidateIntrinsicContentSize`.

There is another approach, but I haven't tried it yet:

```
tableView.selfSizingInvalidation = .enabledIncludingConstraints
```

Source https://appcircle.io/blog/using-swiftui-inside-uicollectionview-and-uitableview

Reply:
I didn't notice any problems about the cell sizing. Did you?

Reply:
Not in this PR, but I'm pretty sure I've tried this approach before and had self-sizing issues. So I'm surprised it's working now. 🤷

Reply:
Resolved https://github.com/wordpress-mobile/WordPress-iOS/pull/22593/commits/ee81dd76422223d721408b45a56df4869cf16611

Reply:
[Nit] `controller.view.layoutIfNeeded()` is no longer needed when `invalidateIntrinsicContentSize` is called.

Reply:
```suggestion
```